### PR TITLE
Add an example for LWR iiwa with Xtion/Kinect

### PR DESCRIPTION
**Pull request by lyh458**
Thank you for your awesome work!
This is an example for LWR iiwa with Xtion/Kinect, which have been verified on real robot.

### CODE REVIEW

File: docs/example_launch/iiwa_kinect_xtion_calibration.launch [hunk 1]
```diff
@@ -0,0 +1,60 @@
+<launch>
+    <arg name="namespace_prefix" default="iiwa_kinect_xtion" />
+    <arg name="marker_size" default="0.05" doc="Size of the ArUco marker used, in meters" />
```
Comment (marcoesposito1988) [inline docs/example_launch/iiwa_kinect_xtion_calibration.launch:3]:
```suggestion
    <arg name="marker_size" doc="Size of the ArUco marker used, in meters" />
```

File: docs/example_launch/iiwa_kinect_xtion_calibration.launch [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+    <arg name="marker_id" default="100" doc="The ID of the ArUco marker used" />
```
Comment (marcoesposito1988) [inline docs/example_launch/iiwa_kinect_xtion_calibration.launch:4]:
```suggestion
    <arg name="marker_id" doc="The ID of the ArUco marker used" />
```

**Review by marcoesposito1988 — CHANGES_REQUESTED** [review]
Hey @lyh458,

thanks a lot, this may be very helpful for new users!

I would suggest to remove the default values for the argument regarding the marker ID and size, because if a new user prints some different marker, this will otherwise silently fail. Removing the default forces the user to enter the correct values. What do you think?

**Comment by lyh458** [comment]
Yes, you are right!

**Review by marcoesposito1988 — APPROVED** [review]

**Comment by marcoesposito1988** [comment]
Great, thanks a lot for your contribution! Further help is welcome anytime :)

**Pull request merged**